src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -98,9 +98,15 @@ public void setReplay(Replay replay) {
         .average()
         .ifPresent(averageRating -> ratingLabel.setText(i18n.number((int) averageRating)));
 
-    durationLabel.setText(Optional.ofNullable(replay.getEndTime())
-        .map(endTime -> timeService.shortDuration(Duration.between(replay.getStartTime(), endTime)))
-        .orElse(i18n.get("notAvailable")));
+    Integer replayTicks = replay.getReplayTicks();
+    if (replayTicks != null) {
+      durationLabel.setText(timeService.shortDuration(Duration.ofMillis(replayTicks * 100)));
+      ((Label) durationLabel.getGraphic()).setText("");
```

Comment:
Maybe we should use the unicode format ala `\u0055`??

Reply:
That's normally what I would do, but it's like this in the rest of the code so I was just trying to keep it consistent.